doc/languages-frameworks/python.section.md

```diff
@@ -1471,9 +1471,11 @@ with python311Packages;
 buildPythonPackage rec {
   name = "mypackage";
   src = ./path/to/package/source;
-  propagatedBuildInputs = [
+  dependencies = [
```

Comment:
I am not super happy with that name. Mainly because it is so different to all the other names we already have. Not sure if pythonDependencies would be any better, that was just the first idea I had right now.

Reply:
maybe buildSystemDependencies? Maybe someone else has a smart idea for something more nix-y.

Reply:
I think `(python)buildSystemDependencies` would be maximally confusing, given that "build-system.requires" in pyproject.toml is something entirely different (see above)

`dependencies` seems perfectly fine to me, mostly because that's what modern python calls it. A python prefix can be seen as implied as its an argument to a function called `buildPythonPackage`.